controller/app/validators/key_content_validator.rb

```diff
@@ -0,0 +1,15 @@
+class KeyContentValidator < ActiveModel::Validator
+   def validate(record)
+     key = Net::SSH::KeyFactory.load_data_public_key(record.type + " " + record.content)
```

Comment:
@sdodson Do you happen to have an example of anything that Net:SSH can't parse? I have been trying various invalid inputs (including just changing the code to pass a random string to load_data_public_key), but in all cases the handling seems to be working OK with a proper error message on the console. I agree that we probably need some exception handling, but would like to be able to find some test cases that I can use to properly test it. Any thoughts?